### ISSUE TITLE
EZfarm is down.. 

### ISSUE THREAD

**Issue by Cptnsavahoe** [EasyFarm/EasyFarm#290]
After todays update from windower it no longer targets and engages mob.

**Comment by Mercurian** [comment]
irritating. Manually revert the hook back to 4.3.2.3 (disconnect internet > open windower > reconnect internet) and it will work for now.

**Comment by YogaHoser** [comment]
I tried what Mercurian suggested but it's still not working for me.

My log shows Now Running and Now Targeting correct mob but no reaction from character.  Won't call trusts or engage.  If I manually engage, JA's & WS's won't be used.

If anyone has other suggestions I would be glad to try them.  Does anyone know if this issue is being worked on?

**Comment by alankidd** [comment]
I am running into the same issue, my character wont Pull or Engage even if i get aggro. Also wont use abilities or WS when I engage manually. It targets monsters but wont do anything else..  Hoping someone finds a fix to this soon!

**Comment by daegon85** [comment]
Just an fyi, works if you use Ashita. Wasn't working for me in windower.

**Comment by Porcius** [comment]
I have had 1 account logged in since before update was pushed and Easy Farm loaded.  That account is fine.  I loaded up another account and Easy Farm is having this issue.   I hope this is a simple fix i am afraid to relog.

**Comment by Mercurian** [comment]
You need to /actually/ revert your hook manually. Back to 4.3.2.3 then disconnect internet, open windower then reconnect internet so that it can't update to the one that breaks easyfarm until its fixed.

**Comment by boleslaus** [comment]
i think widower updated again cause i restarted today and it reinstalled a prev hook and works fine now

**Comment by Atomic646** [comment]
anyone have the previous hook for download or know where to get it?

**Comment by bonz129** [comment]
anyone still having problem with easyfarm ? mine still isnt working =(

**Comment by mbout4it** [comment]
confirmed still  not working

**Comment by Mercurian** [comment]
Um, no, it's been working fine. It was down for less than 24 hours and cair pushed a new hook.

**Comment by yomang** [comment]
Mine still isn't working. I tried reinstalling Windower to a new directory and EF still doesn't engage or move along the path I have set.  If I use the same exact settings and attach to an ashita process, it works perfectly fine as it used to before the hook update 3 weeks ago. Just tried this morning and see hook.dll version 4.3.4.7 and it is still doing the same thing. No action when running EF except once in a while it whips the characters view around and still does nothing.  No idea what I could even do anymore to fix it besides reinstalling FFXI completely. But Ashita works, so I'm thinking that is not going to fix anything.
